concent_api/core/subtask_helpers.py

```diff
@@ -1,9 +1,10 @@
 from base64                     import b64encode
 from logging import getLogger
+from typing import Dict
 from typing import List
 from typing import Optional
 
-from django.db.models           import Q
+from django.db.models import Q
 from django.utils               import timezone
```

Comment:
We decided last time, that if we're changing code next to old style code we should change also block of code where is old style :)